doc/about.rst

```diff
@@ -0,0 +1,47 @@
+.. _about:
+
+======================
+About ``mybinder.org``
+======================
+
+This page explains some of the teams and organizations behind mybinder.org.
+
+What technology runs mybinder.org?
+==================================
+
+The technology behind mybinder.org is primarily composed of three open-source projects:
+
+* `JupyterHub <https://z2jh.jupyter.org>`_, which manages cloud infrastructure for user instances
+* `repo2docker <repo2docker.readthedocs.io>`_, which builds Docker images from GitHub repositories
+* `BinderHub <binderhub.readthedocs.io>`_, which orchestrates the above two projects and
+  provides the Binder interface.
+
+Each of these are open and inclusively-governed projects. Currently, these are all officially
+hosted as a part of Project Jupyter, an open project that creates open tools for data science
```

Comment:
Link to Project Jupyter governance page on jupyter.org

Reply:
I would start with org definition, teams, sponsors (Moore) and end with technical projects. 

Perhaps make  the distinction that all  of the technical components are governed by Project Jupyter under the JupyterHub org. The long term sustainability of mybinder.org is driven by the mybinder.org team who is made up of ...